src/project-css/app/Console/Kernel.php

```diff
@@ -24,8 +24,7 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
-        // $schedule->command('inspire')
-        //          ->hourly();
+      //
```

Comment:
Why is schedule still delcared if there are no methods?

Reply:
Whole Console file look like it could go.  Not sure its needed.

Reply:
removing the /Console/Kernel.php caused errors when running the tests was able to remove the unused schedule function.